.github/workflows/build.yml

```diff
@@ -25,14 +27,14 @@ jobs:
       - name: Install NPM Dependencies
         run: npm ci
 
-      - name: Build Web App
-        run: npx gulp build --platform=browser
-
-      - name: Test Web App
-        run: npm run action scripts/test
-
-      - name: Build Electron App
+      - name: Build Linux Client
         run: npm run action src/electron/package_linux
+      
+      - name: Upload Linux Client
+        uses: actions/upload-artifact@v2
+        with:
+          name: unsigned-linux-client
```

Comment:
I've prefixed these with `unsigned-*` because I anticipate we won't be able to figure out how to sign them all at the same time (see: windows). We can remove the prefix as we add the signing logic.